orbit/pkg/packaging/linux_shared.go

```diff
@@ -364,7 +364,9 @@ pkill fleet-desktop || true
 func writePostRemove(opt Options, path string) error {
 	if err := ioutil.WriteFile(path, []byte(`#!/bin/sh
 
-rm -rf /var/lib/orbit /var/log/orbit /usr/local/bin/orbit /etc/default/orbit /usr/lib/systemd/system/orbit.service /opt/orbit
+if [ "$1" = 0 ]; then
+	rm -rf /var/lib/orbit /var/log/orbit /usr/local/bin/orbit /etc/default/orbit /usr/lib/systemd/system/orbit.service /opt/orbit
+fi
```

Comment:
I'm not too familiar with this part of the code, can you point me to where this script is being called?

Reply:
@mostlikelee hey this is used as a template file for the RPM/DEB package post run script.

We use https://github.com/goreleaser/nfpm to create RPM/DEB installers of `fleetd` so this is getting used during `fleetctl package -type=rpm`

This particular script is executed during the `%postun` phase of a RPM upgrade/uninstall.

The problem here is that when upgrading, I think the `%postun` phase is executed and will remove the binary files. This conditional checks to see if we are actually doing a real uninstall before removing the files.

Reply:
thanks for that! do we need to change where this script is called to pass the new argument, or is that handled natively by the installer?

Reply:
The system package manager is handling passing the args, yum, rpm, etc.

Reply:
LGTM!

Reply:
@mostlikelee I added some details in the issue https://github.com/fleetdm/fleet/issues/14380